src/components/LandingPages/ClientLanding.tsx

```diff
@@ -1,27 +1,143 @@
-import React, { Component } from 'react';
+import React, { Component, CSSProperties } from 'react';
 import { Helmet } from 'react-helmet';
-import { Link, RouteComponentProps, withRouter } from 'react-router-dom';
+import { RouteComponentProps, withRouter } from 'react-router-dom';
 
-import AppSVG from '../../static/images/calendar.svg';
-import FileSVG from '../../static/images/file.svg';
-import SignDoc from '../../static/images/sign-document.png';
+import getServerURL from '../../serverOverride';
+import AtWorkPic from '../../static/images/atwork-rafiki.png';
+import DocumentsPic from '../../static/images/documents-rafiki.png';
 import BaseCard, { CardImageLoc, CardSize } from '../BaseComponents/BaseCard';
 
 interface Props extends RouteComponentProps {
   name: String
+  username: String
 }
 
-class ClientLanding extends Component<Props, {}, {}> {
+interface State {
+  activities: Array<any>,
+  isLoading: Boolean
+}
+
+const activitiesCardStyles: CSSProperties = {
+  border: '1px solid #D3D3D3',
+  backgroundColor: '#F8F8F8',
+  boxSizing: 'border-box',
+  borderRadius: '1.5%',
+  marginBottom: '1.5%',
+  overflow: 'hidden',
+};
+
+const activityTitleStyles: CSSProperties = {
+  float: 'left',
+  color: '#6C757D',
+  fontFamily: 'Raleway',
+  fontStyle: 'normal',
+  fontWeight: 'bold',
+  fontSize: '16px',
+  lineHeight: '18px',
+  padding: '0.75%',
+};
+
+const activityDateStyles: CSSProperties = {
+  float: 'right',
+  color: '#6C757D',
+  fontFamily: 'Inter',
+  fontStyle: 'normal',
+  fontWeight: 'normal',
+  fontSize: '16px',
+  lineHeight: '18px',
+  padding: '0.75%',
+};
+
+interface ActivityProps {
+  activity: any,
+}
+
+function ActivitiesCard(props: ActivityProps) {
```

Comment:
I like that this activities card is in its own functional component. Maybe in the future, we might want to break this out into its own design component- that can be a follow up action item though that doesn't have to be in this PR. It would be even better if it could be an extension of the card design component too.